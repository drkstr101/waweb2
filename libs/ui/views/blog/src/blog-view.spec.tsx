import { render } from '@testing-library/react';

import BlogView from './blog-view';

describe('BlogView', () => {
  it('should render successfully', () => {
    const { container } = render(<BlogView />);
    expect(container).toBeInstanceOf(HTMLElement);
    expect(container).toMatchSnapshot();
  });
});
