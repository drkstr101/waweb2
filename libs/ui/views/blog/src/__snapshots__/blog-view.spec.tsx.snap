// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogView should render successfully 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Welcome to BlogView!
    </h1>
  </div>
</div>
`;
